docs/src/design/chiplets/hasher.md

```diff
@@ -232,49 +235,67 @@ The semantics of `MV` and `MU` instructions are similar to the semantics of `MP`
 When describing AIR constraints, we adopt the following notation: for column $x$, we denote the value in the current row simply as $x$, and the value in the next row of the column as $x′$. Thus, all transition constraints described in this note work with two consecutive rows of the execution trace.
 
 ### Row address constraint
+
 As mentioned above, row address $r$ starts at $1$, and is incremented by $1$ with every row. The first condition can be enforced with a boundary constraint which specifies $r=1$ at the first row. The second condition can be enforced via the following transition constraint:
 
->$$
-r' - r - 1 = 0  \text{ | degree } = 1
+> $$
+> r' - r - 1 = 0  \text{ | degree } = 1
+> $$
+
 $$
```

Comment:
There still seem to be some newlines here (and in other places) which mess up formula rendering.